client/src/app/components/stardust/Asset.tsx

```diff
@@ -1,57 +1,43 @@
-import React, { Component, ReactNode } from "react";
+import { FunctionComponent } from 'react';
```

Comment:
Lets just use double-quotes " in imports everywhere.

Reply:
Done